packages/table/src/types.ts

```diff
@@ -74,9 +76,16 @@ declare module 'react-table' {
   export type PluginHook<TData extends TableData> = PluginHookDefault<TData>;
 
   // @ts-ignore
-  export interface TableState {
+  export interface TableState<TData extends TableData = TableData> extends Partial<UsePaginationState<TData>> {
     menu: MenuState;
   }
+
+  // @ts-ignore
+  export interface TableInstance<D extends TableData> extends UsePaginationInstanceProps<D> {
+    disablePagination?: boolean;
+    pageSizes?: number[];
```

Comment:
Should these have some documentation? 
e.g. 
```ts
/* The page size options shown inside the select of pagination*/
pageSizes?: number[];
```

Reply:
sure, easier for dev to know what it is since it will show as highlight